docs/cpp-runnable.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Generating Executables
+id: cpp-runnable
+---
+
+The `fuse` CLI provides the `run` subcommand for generating executables that
+can be used to test Fuse code without going throught a synthesis workflow.
+
+To build an executable, first run the following command from the project root:
+
+```
+sbt assembly
+```
+
+This will rebuild the compiler and download any headers required for building
+the executable.
```

Comment:
Everyone will have already done that in order to use the compiler at all, right?